harmony_netcdf_to_zarr/convert.py

```diff
@@ -176,6 +175,7 @@ def _output_worker(output_queue: Queue, shared_namespace: Namespace,
             break
 
         try:
+            logger.info('processing granule')
```

Comment:
I had that before, but all it shows is /tmp/fjdaj8eohaihfhgiuerh.hdf as the input_granule since it's the downloaded granule that is being used and there isn't an easy access to any granule id or human useful information.

Reply:
Ohhhh. Yeah - good point. The downloaded file name is a randomised string, not the actual input file name. Yeah, in that case, no worries. (Unless you wanted to try adding a shared integer as a counter... but there's probably enough work in this PR already)

Reply:
Opted not to try to share across multiprocess so each one counts up.  9e8bc1e